docs/Drawer.md

```diff
@@ -90,17 +90,33 @@ The `Drawer` has four `variant`s:
 
 > **Note on using multiple drawers**: If your application uses multiple `<Drawer>`s, each `<DrawerLayout>` will automatically adjust based on the state of the nearest `<Drawer>`. If you are using a `<Drawer>` without a `<DrawerLayout>`, you should set the `noLayout` property to true on the `<Drawer>` to prevent inadvertently affecting the styles of any `<DrawerLayout>`s.
 
-#### Classes
+### Style Overrides
+
+You can override the default styles used by Brightlayer UI by:
+
+-   using the `sx` prop
+-   passing a `classes` prop with keys from the `Name` column below
+-   using the `Global CSS Class` in your main stylesheet
+
+For more details on styling options check out our [Styling Guide](https://github.com/brightlayer-ui/react-component-library/tree/master/docs#style-guide).
+
+| Name       | Global CSS Class       | Description                                                    |
+| ---------- | ---------------------- | -------------------------------------------------------------- |
+| root       | .BluiDrawer-root       | Styles applied to the root element                             |
+| content    | .BluiDrawer-content    | Styles applied to the drawer content container                 |
+| expanded   | N/A*                   | Styles applied to the root element when the drawer is expanded |
+| paper      | .BluiDrawer-paper      | MUI Drawer style override for the underlying paper element     |
+| sideBorder | .BluiDrawer-sideBorder | Styles applied when sideBorder is set to true                  |
 
-You can override the classes used by Brightlayer UI by passing a `classes` prop. The Drawer supports the following keys:
+> \* This class cannot be overridden via the `sx` prop. However, the `.Mui-expanded` class can be used to override expanded styles with sx.
 
-| Name       | Description                                                    |
-| ---------- | -------------------------------------------------------------- |
-| root       | MUI Drawer style override for the root element                 |
-| content    | Styles applied to the drawer content container                 |
-| expanded   | Styles applied to the root element when the drawer is expanded |
-| paper      | MUI Drawer style override for the underlying paper element     |
-| sideBorder | Styles applied when sideBorder is set to true                  |
+```tsx
+import { Drawer } from '@brightlayer-ui/react-components';
+...
+<Drawer variant='temporary' sx={{margin: '4px'}}
+>
+</Drawer>
+```
```

Comment:
Why is this example here? The sections usually go Description, Usage/Example, API, Style Overrides (and there's already an example above).